README.md

```diff
@@ -2,7 +2,7 @@
 
 ![Requires Xcode 12](https://img.shields.io/badge/Xcode-12-1575F9.svg?style=flat&logo=xcode&logoColor=1575F9)
```

Comment:
This was true at the time (Circa spring 2021).  We would only need to keep this up to date when our CI changes.